src/Chainweb/Version/RecapDevelopment.hs

```diff
@@ -69,6 +69,7 @@ recapDevnet = ChainwebVersion
             Chainweb221Pact -> AllChains $ ForkAtBlockHeight $ BlockHeight 580
             Chainweb222Pact -> AllChains $ ForkAtBlockHeight $ BlockHeight 590
             Chainweb223Pact -> AllChains $ ForkAtBlockHeight $ BlockHeight 600
+            Chainweb224Pact -> AllChains ForkNever
```

Comment:
For test versions and recap-devnet, go ahead and add a height, following the patterns that are already there.